includes/wc-product-functions.php

```diff
@@ -982,7 +982,14 @@ function wc_get_price_including_tax( $product, $args = array() ) {
 				$taxes_total = array_sum( array_map( 'wc_round_tax_total', $taxes ) );
 			}
 
-			$return_price = round( $line_price + $taxes_total, wc_get_price_decimals() );
+			/**
+			 * If the customer is excempt from VAT, remove the taxes here.
+			*/
+			if ( ! empty( WC()->customer ) && WC()->customer->get_is_vat_exempt() ) { // 
```

Comment:
@stefanstatescu1 Thanks for submitting this PR. This check would work better if it were the condition before `if ( 'yes' === get_option( 'woocommerce_tax_round_at_subtotal' ) ) {` and set `$taxes_total = 0.00`. That way we would avoid calculating the tax total when it wasn't used. The PR also has coding standards issues.

Reply:
Thanks @rrennick 
I think it's better now?